src/htool/solver/solver.hpp

```diff
@@ -48,7 +49,10 @@ void declare_DDM(py::module &m, const std::string &className) {
                 // LCOV_EXCL_STOP
             }
 
-            self.solve(b.data(), x.mutable_data(), mu);
+            {
+              py::scoped_ostream_redirect stream(std::cout, py::module_::import("sys").attr("stdout"));
```

Comment:
Just to be sure I understand, this it to redirect the output to the correct python output so that you can parse it on your side?

Reply:
That's correct. I have a matching change for PyNucleus.